### PR TITLE
Refactor some variables collide with buildin func

### LINKED ISSUE #41: Some variables collide with buildin func
We'd better rename these variables  to avoid unnecessary troubles. Such as cap/copy/new and so on.

/kind cleanup

### PR DESCRIPTION
**Pull request by jiwq**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Refactor some variables collide with buildin func, we should refactor it to avoid some troubles.

e.g. cap new copy and so on.

#### Which issue(s) this PR fixes:
Fixes #41

### CODE REVIEW

**Comment by ahg-g** [comment]
nice cleanup, I will approve once the tests pass.

File: pkg/capacity/snapshot.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func (c *Cache) Snapshot() Snapshot {
 // Snapshot creates a copy of Capacity that includes references to immutable
 // objects and deep copies of changing ones. A reference to the cohort is not included.
 func (c *Capacity) snapshot() *Capacity {
-	copy := &Capacity{
+	cc := &Capacity{
```
Comment (alculquicondor) [inline pkg/capacity/snapshot.go:52]: `capCopy`

File: pkg/scheduler/scheduler.go [hunk 2]
```diff
@@ -93,8 +93,8 @@ func (s *Scheduler) schedule(ctx context.Context) {
 		if usedCapacity.Has(e.Capacity) {
 			continue
 		}
-		cap := snapshot.Capacities[e.Capacity]
-		if len(e.borrows) > 0 && cap.Cohort != nil && usedCohorts.Has(cap.Cohort.Name) {
+		c := snapshot.Capacities[e.Capacity]
```
Comment (alculquicondor) [inline pkg/scheduler/scheduler.go:96]: snapCap

File: pkg/scheduler/scheduler.go [hunk 3]
```diff
@@ -139,13 +139,13 @@ func calculateRequirementsForAssignments(log logr.Logger, workloads []workload.I
 	entries := make([]entry, 0, len(workloads))
 	for _, w := range workloads {
 		log := log.WithValues("queuedWorkload", klog.KObj(w.Obj), "capacity", w.Capacity)
-		cap := snap.Capacities[w.Capacity]
-		if cap == nil {
+		c := snap.Capacities[w.Capacity]
```
Comment (alculquicondor) [inline pkg/scheduler/scheduler.go:142]: snapCap

File: pkg/scheduler/scheduler_test.go [hunk 4]
```diff
@@ -676,8 +676,8 @@ func TestSchedule(t *testing.T) {
 			// Verify assignments in capacity cache.
 			gotAssignments := make(map[string]kueue.QueuedWorkloadSpec)
 			snapshot := capCache.Snapshot()
-			for cName, cap := range snapshot.Capacities {
-				for name, w := range cap.Workloads {
+			for cName, c := range snapshot.Capacities {
```
Comment (alculquicondor) [inline pkg/scheduler/scheduler_test.go:679]: sCap

**Pull request merged**